OtlSync.pas

```diff
@@ -380,6 +367,22 @@     TAutoUnlock = class(TInterfacedObject, IOmniLockManagerAutoUnlock)
     function  BaseEvent: TEvent;
   end;
 
+  IOmniCancellationToken = interface ['{5946F4E8-45C0-4E44-96AB-DBE2BE66A701}']

```

Comment:
IOmniCancellationToken must exist in Delphi 2007 (where OTL_Generics is not defined).